src/components/DocumentStatus/DocumentStatus.tsx

```diff
@@ -36,10 +36,13 @@ export const IssuedBy: FunctionComponent<DocumentStatusProps> = ({ verificationS
 
   const identityProofFragment = verificationStatus.find(
     (status) => status.type === "ISSUER_IDENTITY" && status.status === "VALID"
-  ) as VerificationFragment;
+  ) as VerificationFragmentWithData;
```

Comment:
have a look over the whole repo to identify places we can use the utility functions from oa-verify instead of maintaining separate logic here

https://github.com/Open-Attestation/oa-verify/blob/f10e87e9c2515059e1c5eb661d12655111695170/README.md#L213-L214

Reply:
looked through the repo, only place is the fragments.tsx file that i dont think can refactor. please advice on that file.